cmd/neofs-node/config/engine/config.go

```diff
@@ -47,9 +47,9 @@ func IterateShards(c *config.Config, required bool, f func(*shardconfig.Config))
 	}
 }
 
-// ShardPoolSize returns value of "shard_pool_size" config parameter from "storage" section.
+// ShardPoolSize returns the  value of "shard_pool_size" config parameter from "storage" section.
```

Comment:
```suggestion
// ShardPoolSize returns the value of "shard_pool_size" config parameter from "storage" section.
```
here and below (think could be fixed with regex or in IDE)

Reply:
Checked everywhere